src/components/header/header.yaml

```diff
@@ -476,3 +476,120 @@ examples:
     menuButtonText: 'Custom button text'
     menuButtonLabel: 'Custom button label'
     navigationLabel: 'Custom navigation label'
+- name: with custom navigation items and a sign out link
```

Comment:
might be worth having two extra test cases in VRT (or trying it out to check)
- one with less nav items
- and one with enough to have multiple lines

Reply:
I don't actually know how multiple lines will look

Reply:
it might also be good to split this into two commits, so we can see the changes before and after

basically could just undo the _header.css change on this commit and redo it in a new commit - where one before has the addition of the new backstop scenarios (with issue visible) and then new commit has the fixes and we can compare the changes between them

Reply:
I have added two more examples

_with service name and one custom navigation item and a sign out link
with service name and lots of custom navigation items and a sign out link_